src/exchange.js

```diff
@@ -65,6 +65,16 @@ class Exchange {
     return this._QuoteClass.getQuote(this._api, this._delegate, -amount, baseCurrency, quoteCurrency, this._debug);
   }
 
+  getSellQuote (amount, baseCurrency, quoteCurrency) {
```

Comment:
Looks like this function is just copied from `getBuyQuote`, could you remove the duplication here? Correct me if I'm wrong, but I think this could be rewritten as just:

```
getSellQuote (amount, baseCurrency, quoteCurrency) {
  return getBuyQuote(-amount, baseCurrency, quoteCurrency);
}
```